superduperdb/jobs/job.py

```diff
@@ -246,7 +290,72 @@ def dict(self):
                 'method_name': self.method_name,
                 'component_identifier': self.component_identifier,
                 'type_id': self.type_id,
-                'cls': 'ComponentJob',
+                '_path': 'superduperdb/jobs/job/ComponentJob',
             }
         )
         return d
+
+
+def remote_job(identifier, dependencies=()):
+    """
+    Remote Job to execute remote tasks.
+
+    :param identifier: Job identifier.
+    :param dependencies: List of dependencies.
+    """
+    # Connect to remote cluster
+    from superduperdb import CFG
+    from superduperdb.base.build import build_compute
+
+    compute = build_compute(CFG.cluster.compute.uri)
+
+    assert compute.remote is True, "Compute is not a distributed backend type."
+
+    compute.execute_task(identifier, dependencies=dependencies)
+
+
+def remote_task(identifier, dependencies=()):
+    """
+    Load job from job metadata and schedule it on remote cluster.
+
+    :param identifier: Job identifier.
+    :param dependencies: List of dependencies.
+    """
+    from superduperdb import CFG
+    from superduperdb.base.build import build_datalayer
+
+    # TODO: Make this run a predict job with multiple
+    # chunks as tasks.
+    db = build_datalayer(CFG, cluster__compute__uri=None)
+
+    info = db.metadata.get_job(identifier)
+
+    args = info['args']
+    kwargs = info['kwargs']
+    path = info['_path']
+
+    # TODO: Add compute kwargs.
+    if 'ComponentJob' in path:
+        component_identifier = info['component_identifier']
+        method_name = info['method_name']
+        type_id = info['type_id']
+        job = ComponentJob(
+            args=args,
+            kwargs=kwargs,
+            method_name=method_name,
+            component_identifier=component_identifier,
+            type_id=type_id,
+            db=db,
+        )
+    elif 'FunctionJob' in path:
+        import importlib
+
+        function = path.split('/')[-1]
+        import_path = FunctionJob._registered_function[function]
```

Comment:
Why not just import the function from the path instead of this `_registered_function` boilerplate?